ui/app/selectors/selectors.js

```diff
@@ -14,6 +14,16 @@ export function getNetworkIdentifier (state) {
   return nickname || rpcUrl || type
 }
 
+export function getCurrentNetworkName (state) {
```

Comment:
Looking at the `getNetworkIdentifier` selector above, the naming here is a bit confusing.

Can we rename this to `getMetricsNetworkIdentifier` to better indicate its intended use?